### DISCUSSION TITLE
MD type error

### DISCUSSION THREAD

**Discussion by dvnrsn** [pngwn/MDsveX#369] [Q&A]
New sveltekit skeleton app
```
Cannot find module '*.md' or its corresponding type declarations
```
I can resolve this by [declaring an MD module in globals.d.ts](https://stackoverflow.com/a/56659180) but wanted to make sure that's the correct approach.

**Comment by outerlook** [comment]
I'm on this too. To extend it: should typescript already be infering correct typings for my components with it? Here it isn't.
I'm trying to use MDsveX to compile email components on my backend using svelte-mail, svelte-esbuild and windicss. I think it is an interesting stack to use with MDsveX, but I'd really like some type safety.

And if I import from `fileame.svx` I get module not found error, but from `filename` without extension works.

**Comment by KianYang-Lee** [comment]
I think this is the correct approach? Specifically, `globals.d.ts` should be declared in `src/` folder
